test/gcs/cri_test.go

```diff
@@ -0,0 +1,95 @@
+//go:build linux
+
+package gcs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Microsoft/hcsshim/internal/guest/prot"
+	"github.com/Microsoft/hcsshim/internal/guest/stdio"
+)
+
+//
+// tests for operations on sandbox and workload (CRI) containers
+//
+
+func TestCRILifecycle(t *testing.T) {
```

Comment:
Same thing here about adding a comment for what this function is expected to do and what its expected to setup, etc